### DISCUSSION TITLE
Unix timestamp integration

### DISCUSSION THREAD

**Discussion by XenonKaguno** [misskey-dev/misskey#15068] [Ideas]
I know this is a small thing but I would Heavily appreciate it if you integrated the ability to use Unix timestamps within webpages
In discord your able to do <t:1732787869> for example and it would auto convert that to a time in everyones timezone, It would be a heavily welcomed feature 🙏

**Comment by syuilo** [comment]
Use `$[unixtime 1732787869]`

**Reply by XenonKaguno** [reply]
Thank you!!!
